packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -92,6 +92,8 @@ export class DocumentDeltaConnection
 		return this._disposed;
 	}
 
+	public readonly supportDeltaConnectionUpdates = true;
```

Comment:
I assume "true" here does not mean we actually have extra info (in case of R11s), it's just that we can register for events, right?

Reply:
Yes

Reply:
Removed it since no longer needed after emitting event on service.